css/style.css

```diff
@@ -117,9 +119,9 @@ body {
 }
 
 .f-section {
-	height: 95vh;
+	height: 88vh;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_A/pullRequest?prid=6206892)